intellij-plugin-structure/structure-intellij/src/main/java/com/jetbrains/plugin/structure/intellij/plugin/IdePluginManager.kt

```diff
@@ -86,12 +89,13 @@ class IdePluginManager private constructor(
   }
 
    private fun loadPluginInfoFromDirectory(
-    pluginDirectory: Path,
-    descriptorPath: String,
-    validateDescriptor: Boolean,
-    resourceResolver: ResourceResolver,
-    parentPlugin: PluginCreator?,
-    problemResolver: PluginCreationResultResolver
+     pluginDirectory: Path,
+     descriptorPath: String,
+     validateDescriptor: Boolean,
+     resourceResolver: ResourceResolver,
+     parentPlugin: PluginCreator?,
+     problemResolver: PluginCreationResultResolver,
+     parsingConfiguration: PluginParsingConfiguration
```

Comment:
do we need to pass through PParsConfig everywhere here? it's a property of IdePluginManager